tests/resources/VCH-Util.robot

```diff
@@ -128,6 +128,54 @@ Get Docker Params
     Run Keyword If  ${port} == 2376  Set Environment Variable  VCH-PARAMS  -H ${dockerHost} --tls
     Run Keyword If  ${port} == 2375  Set Environment Variable  VCH-PARAMS  -H ${dockerHost}
 
+Get Compose Params  ${output}  ${certs}  ${dockerHost}
+
+Get Compose Params
```

Comment:
My intent was to just run a specific CI test for the docker-compose with Loc's TLS changes, planning to only check this in(only the robot and yml's NOT VCH-Util) after Loc's changes are merged in. 

This also needs some test changes as @chengwang86 suggested depending on what is supported by docker-compose at the present moment.